src/ripple/protocol/impl/LedgerFormats.cpp

```diff
@@ -101,13 +101,11 @@ LedgerFormats::LedgerFormats ()
             ;
 
     add ("Amendments", ltAMENDMENTS)
-            << SOElement (sfLedgerSequence,      SOE_OPTIONAL)
```

Comment:
Just a question.  I'm assuming that it's okay to remove these fields because a) they were optional and b) we're very confident there are no currently existing ledgers with these fields in them.  Or is there yet more I need to learn?  Thanks.